public/app/core/components/Breadcrumbs/utils.ts

```diff
@@ -54,7 +51,8 @@ export function buildBreadcrumbs(sectionNav: NavModelItem, pageNav?: NavModelIte
     addCrumbs(pageNav);
   }
 
-  addCrumbs(sectionNav);
+  // shouldDedupe = true enables app plugins to control breadcrumbs of their root pages
+  addCrumbs(sectionNav, true);
```

Comment:
@ashharrison90 not sure this works, as plugins never send sectionNav, only pageNav.  (see PluginPage)

What if we only dedupe if `isSameNodeText && isSamePathAsLastBreadcrumb`  ?

Reply:
@ashharrison90 I tested this branch and it still works like it should!

Reply:
perfect, time to raise a PR then and check it still works for gilles as well 😄